### DISCUSSION TITLE
Skywalking Trace table view does not show data

### DISCUSSION THREAD

**Discussion by asasas234** [apache/skywalking#8521] [Q&A]
<img width="1886" alt="image" src="https://user-images.githubusercontent.com/4795442/153132964-72946e41-66bd-42e1-86c9-141602997889.png">
<img width="1913" alt="image" src="https://user-images.githubusercontent.com/4795442/153133019-1711e317-12e6-45fe-84ab-66e974196f05.png">

The table view has no data, but the statistics, tree structure and lists all have data

I can see on the console that when I check the table, the data is there, but for some reason it doesn't show up

skywalking-oap version is 8.8.1

The specific Json returned is attached for local reproduction of the problem.

[json.txt](https://github.com/apache/skywalking/files/8029980/json.txt)
m

**Comment by asasas234** [comment]
No errors reported in the web console

**Comment by wu-sheng** [comment]
I am afraid we will not response this soon. It is better you could debug by yourself. Rocketbot UI is going to be replaced by booster UI in the next 2 months. So, all UI related issues would be pushed back
